test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/Helpers.cs

```diff
@@ -10,7 +10,7 @@ public class Helpers
     {
         public static string GetTestSitesPath(ApplicationType applicationType)
         {
-            return Path.GetFullPath(Path.Combine("..", "..", "..", "..", "..", applicationType == ApplicationType.Standalone ? "TestSites" : "TestSites.Portable"));
+            return Path.GetFullPath(Path.Combine(Directory.GetCurrentDirectory(), "..", applicationType == ApplicationType.Standalone ? "TestSites" : "TestSites.Portable"));
```

Comment:
Do you even need the `Directory.GetCurrentDirectory()` here? It was not there before